tollway/vehicle.py

```diff
@@ -42,7 +43,7 @@ def get_tollways(html: Optional[str] = None) -> dict:
             if sibling.name == "table":
                 tolls.extend(
                     [
-                        td.text.strip()
+                        unicodedata.normalize("NFKD", td.text.strip())
```

Comment:
nice find, I think we have a pybites tip as well about this, haven't touched this in ages though :)
how did you find this?

Reply:
I would like to say GPT but found it on stack overflow: https://stackoverflow.com/questions/10993612/how-to-remove-xa0-from-string-in-python (2nd answer)